corehq/feature_previews.py

```diff
@@ -100,7 +100,8 @@ def previews_enabled_for_domain(domain):
     description=_(
         "Specify a custom xpath expression to calculate a value "
         "in the case list or case detail screen."),
-    help_link='https://confluence.dimagi.com/display/commcarepublic/Calculations+in+the+Case+List+and+Details'
+    help_link=('https://dimagi.atlassian.net/wiki/spaces/commcarepublic'
```

Comment:
More specific: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143951603/Case+List+and+Case+Detail+Configuration#Calculations-in-the-Case-List%2FDetail